pkg/metrics/metrics.go

```diff
@@ -66,29 +80,33 @@ func HandleTracking(cmd *cobra.Command, commandPath string, app *application.Ava
 	if !userIsOptedIn(app) {
 		return
 	}
-	if !cmd.HasSubCommands() && CheckCommandIsNotCompletion(cmd) {
-		trackMetrics(app, commandPath, flags)
+	if !app.Cmd.HasSubCommands() && CheckCommandIsNotCompletion(app.Cmd.CommandPath()) {
+		trackMetrics(app, flags, err)
 	}
 }
 
-func CheckCommandIsNotCompletion(cmd *cobra.Command) bool {
-	result := strings.Fields(cmd.CommandPath())
+func CheckCommandIsNotCompletion(commandPath string) bool {
+	result := strings.Fields(commandPath)
 	if len(result) >= 2 && result[1] == "completion" {
 		return false
 	}
 	return true
 }
 
-func trackMetrics(app *application.Avalanche, commandPath string, flags map[string]string) {
+func trackMetrics(app *application.Avalanche, flags map[string]string, cmdErr error) {
 	if telemetryToken == "" {
 		telemetryToken = os.Getenv(constants.MetricsAPITokenEnvVarName)
 	}
+	if telemetryToken == "" && !utils.IsE2E() {
+		app.Log.Warn("no token is configured for sending metrics")
+	}
```

Comment:
do we need to users to see this?

Reply:
this is only going into avalanche.log, not stdout. it is useful for debugging. not expected to be printed on releases.